io.catenax.secondary_material_content/1.0.0/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release"}
+{ "status" : "deprecated"}
```

Comment:
a small correction required. this should be '`deprecate`', without 'd'

Reply:
Done